modules/using-assisted-installer-oci-agent-iso.adoc

```diff
@@ -58,10 +63,5 @@ You can create an SSH authentication key pair by using the `ssh-keygen` tool.
 
 .. Download the file to your local system.
 
-. Upload the discovery ISO image to the bucket by completing the steps in the link:https://docs.public.oneportal.content.oci.oraclecloud.com/en-us/iaas/Content/Object/Tasks/managingobjects_topic-To_upload_objects_to_a_bucket.htm[Uploading an Object Storage Object to a Bucket] section of the Oracle documentation.
-
-. From the *Objects* menu, locate your uploaded image, and then click the overflow menu.
-
-. From the *Create Pre-Authenticated Request* window, select the *Object* tile.
-
-. After you create the request, copy the URL from the *Pre-Authenticated Request URL* field. See link:https://docs.oracle.com/en-us/iaas/Content/Object/Tasks/usingpreauthenticatedrequests_topic-To_create_a_preauthenticated_request_for_all_objects_in_a_bucket.htm[Creating a Pre-Authenticated Request] in the Oracle documentation.
+. Upload the discovery ISO image to the {oci} bucket. See link:https://docs.oracle.com/iaas/Content/openshift-on-oci/installing-assisted.htm#install-cluster-upload-iso[Uploading The Red Hat ISO Image to Object Storage (Oracle documentation)].
+.. You must create a pre-authenticated request for your uploaded discovery ISO image. Ensure that you copy the URL from the pre-authenticated request, because you must specify the URL at a later stage when you create an {oci} stack.
```

Comment:
```suggestion
.. You must create a pre-authenticated request for your uploaded discovery ISO image. Ensure that you make note of the URL from the pre-authenticated request, because you must specify the URL at a later stage when you create an {oci} stack.
```